tests/collections/asr/test_asr_datasets.py

```diff
@@ -341,7 +341,6 @@ def test_dali_char_vs_ref_dataset(self, test_data_dir):
 
         num_samples = 10
         batch_size = 1
-        device = 'gpu' if torch.cuda.is_available() else 'cpu'
```

Comment:
Why this is reverted?

Reply:
That device was not used in the code at all.

Reply:
Lol ok then